fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationOptionsDialog.kt

```diff
@@ -73,7 +74,10 @@ fun TranslationOptionsDialog(
                     modifier = Modifier
                         .fillMaxWidth()
                         .padding(start = 56.dp),
-                    onClick = { onTranslationSettingsClicked() },
+                    onClick = {
+                        Events.translationsAction.record(Events.TranslationsActionExtra("global_settings"))
```

Comment:
I think that the Compose view should not have GleanMetrics and Nimbus dependencies. 
Events.translationsAction.record(Events.TranslationsActionExtra("global_settings")) should be added in the Fragment where the callback onTranslationSettingsClicked() is implemented.

Reply:
Thanks, moved around Nimbus and Glean requests!